mapie/tests/test_conformity_scores.py

```diff
@@ -1,35 +1,55 @@
 import numpy as np
 import pytest
 
-from mapie._typing import ArrayLike, NDArray
+from mapie._typing import NDArray, ArrayLike
 from mapie.conformity_scores import (AbsoluteConformityScore, ConformityScore,
                                      GammaConformityScore)
+from mapie.regression.estimator import EnsembleRegressor
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import KFold
 
-X_toy = np.array([0, 1, 2, 3, 4, 5]).reshape(-1, 1)
-y_toy = np.array([5, 7, 9, 11, 13, 15])
-y_pred_list = [4, 7, 10, 12, 13, 12]
-conf_scores_list = [1, 0, -1, -1, 0, 3]
-conf_scores_gamma_list = [1 / 4, 0, -1 / 10, -1 / 12, 0, 3 / 12]
```

Comment:
QUESTION: The results should not be modified, as different results suggest that the code does not behave in the same way with your modifications. What is the reason for these modifications?

Reply:
These modifications were done when `get_signed_conformity_scores` was taking `estimator` in parameters. I couldn't create an estimator that predicted the exact values of `y_pred` proposed in the files.

Reply:
`get_signed_conformity_scores` does not take `estimator` as a parameter, does it?

Reply:
Not anymore

Reply:
What is the other reason for these changes? If there is none, we need to go back to the previous results.

Reply:
Done